solar-land/scss/_typography.scss

```diff
@@ -0,0 +1,35 @@
+// Typography for Quicksand font
```

Comment:
I think, because the file name is typography so it's clear it's for typography, right?, so this comment is not useful